packages/trader/src/Modules/Contract/Components/AccumulatorsStats/accumulators-stats.tsx

```diff
@@ -1,30 +1,38 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import { DesktopWrapper, Icon, MobileDialog, MobileWrapper, Text } from '@deriv/components';
 import { isDesktop, isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-import { connect } from 'Stores/connect';
+import { connect } from '../../../../Stores/connect';
 import ExpandedTicksHistory from './expanded-ticks-history';
 import TicksHistoryCounter from './ticks-history-counter';
 import { AccumulatorsStatsManualModal } from './accumulators-stats-manual-modal';
 import 'Sass/app/modules/contract/accumulators-stats.scss';
+import { useStore } from '@deriv/stores';
+import { TCoreStores } from '@deriv/stores/types';
 
+type TAccumulatorStats = {
+    is_dark_theme?: ReturnType<typeof useStore>['ui']['is_dark_mode_on'];
+    is_expandable?: boolean;
+    ticks_history_stats: {
+        ticks_stayed_in?: ReturnType<typeof useStore>['trade']['tick_history_stats']['ticks_stayed_in'];
```

Comment:
@henry-deriv  The most important thing is that we need to avoid any dependencies from external types here. Please declare these types directly here. 🙂